internal/functions/efgs/database/database.go

```diff
@@ -122,6 +125,19 @@ func (db Connection) RemoveDiagnosisKey(keys []*efgsapi.DiagnosisKeyWrapper) err
 	return nil
 }
 
+//RemoveOldKeys Removes keys older than date provided as parameter.
+func (db Connection) RemoveOldKeys(dateFrom string) error {
```

Comment:
Sorry, but shouldn't the param be named more like `date_until`, or `older_than_date` or something?